### ISSUE TITLE
Request for Access to Training Data and Missing Files

### ISSUE THREAD

**Issue by GQ-ZHU** [tingxiecsu/CSU-MS2#3]
Hello, thank you for your excellent work and for releasing the code of this project.

I am currently trying to reproduce the training and inference pipeline.
During the process, I found that some key data files appear to be missing from the repository: reference_dataset.csv, collision_energy.csv
<img width="758" height="161" alt="Image" src="https://github.com/user-attachments/assets/ae101a88-6dd1-43c1-99f3-032f14e135cf" />
These files appear to be essential for the inference process to function correctly.However, I cannot find these files in the codebase, nor can I find instructions on how to obtain them.

I checked the csu_ms2_db.db file and noticed that it contains several datasets, but it seems that the pre-training dataset and fine-tuning dataset used in the paper/code may not be included there.Additionally, I noticed that both the codebase and the .db file frequently mention “ConSS”.May I ask whether ConSS was the previous name of the CSU-MS2 project, or does it refer to an earlier version of the data/model pipeline?

Thank you very much for your time and the valuable contribution of this project!
Best regards

**Comment by tingxiecsu** [comment]
Hi, thanks again for your interest in the project and for your detailed questions!

**1. About the missing datasets**
In our paper, the NIST23 database was used for both the fine-tuning stage and the test set evaluation. Since NIST23 is not open-source, we are not allowed to redistribute it. This is why those files are not included in the repository.
You correctly pointed out that the reference structure library was missing. We have now uploaded: data/reference.csv This corresponds to the 1-million-scale structure search library used in the paper.
It is required when you want to run inference on your own unknown MS/MS data (.mgf).

**2. About the pre-training dataset**
The pre-training dataset is extremely large (predicted spectra), so we will host it on Zendo instead.
You can download it here:
https://zenodo.org/records/17756840

**3. About “ConSS”**
Yes — ConSS was the earlier internal name of what later became the CSU-MS2 project.
Some legacy variable names and database fields still use the old name, but they refer to the same pipeline.

If you have any further questions or need help running the training/inference workflow, feel free to ask.
Thanks again for your interest and support!
